doc/modules/user-manual/pages/support/cookbook/index.adoc

```diff
@@ -0,0 +1,11 @@
+= Cookbook
```

Comment:
Not clear for me the difference between cookbook & faq?

Reply:
Both existed for Sirius Desktop (https://wiki.eclipse.org/Sirius/Cookbook vs https://wiki.eclipse.org/Sirius/FAQ).
I've merged both now